Fixed kickoff_play.py MyPy Errors

gordonm867 opened the pull request:
I fixed MyPy errors in kickoff_play.py by replacing "RoleResult" with "role.RoleResult", replacing "calculations" with "wall_calculations" to match an existing class, and modifying the specified return types of some methods to match their actual return types.  I can confirm that the kickoff play still runs on my machine with this code and reports no MyPy errors within the kickoff_play.py file, but I had to change the IPlay class to avoid type incompatibility, and I am not sure if that is what I was supposed to do or if that will destroy other things.

gordonm867 committed:
Fixed errors from MyPy in kickoff_play.py and matched return types in super class.

gordonm867 committed:
Reverted accidental change to main gameplay file that was meant for testing kickoff play but should not be used as competition code.

kfu02 reviewed (changes requested):
Resolve the two style checks complaining about "line too long". (See Files Changed tab.)

Not your fault because the style checks didn't pop up until I manually clicked run on them (link your account with my.robojackets.org to resolve this, I think). I will review the change to the IPlay class after style is fixed.

gordonm867 committed:
Fixed kickoff_play.py line too long

Line 145 was 81 characters, so I split it into two lines within the parenthesis as preferred by Python's style guide.

gordonm867 committed:
Fixed too long line in __init__.py (IPlay class)

Line 140 was 88 characters, so I split it at a comma within brackets as preferred by Python's style guide.

gordonm867 commented:
> Resolve the two style checks complaining about "line too long". (See Files Changed tab.)
> 
> Not your fault because the style checks didn't pop up until I manually clicked run on them (link your account with my.robojackets.org to resolve this, I think). I will review the change to the IPlay class after style is fixed.

Oh, thanks for letting me know.  I should have resolved those issues.  I also connected my account with my.robojackets.org.

gordonm867 committed:
Removed trailing whitespace

On line #140, there was a trailing whitespace after a comma that I removed.

gordonm867 committed:
Removed trailing whitespace

I removed a trailing whitespace following a comma at the end of line #145.

oddlyspookycherry reviewed (approved):
Works well on my end, good job!

kfu02 reviewed (approved):
Nice job!
